build.xml

```diff
@@ -66,17 +85,41 @@
 
     <target name="build"
             description="Compiles all sources to the classes directory">
-	<javac includeantruntime="false" deprecation="yes"
+	    <javac includeantruntime="false" deprecation="yes"
 			source="11" target="11"
-			optimize="true" srcdir="tests"
+			optimize="true" srcdir="tests/api"
 			classpath="${class.path}" destdir="${class.dir}"
 			debug="on" />
- 
+        <javac includeantruntime="false" deprecation="yes"
+            source="11" target="11"
```

Comment:
can you replace this by `release=11` where applicable, please?

Reply:
done